admin/tools/docker/deployment/parallel/run-large-scale-tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Launch Qserv multinode tests
+
+# @author Fabrice Jammes SLAC/IN2P3
+
+set -e
+
+DIR=$(cd "$(dirname "$0")"; pwd -P)
+. "${DIR}/env.sh"
+
+TEST_DIR="/tmp/qserv_testscale"
+rm -rf "$TEST_DIR"
+GIT_REF="tickets/DM-8196"
```

Comment:
Ok with all your remarks, thanks! About separation of qserv and deployment code, I think we could target that in the middle-term, but I still performs some update to configuration code so this is convenient to keep it together in the short-term.